compiler/ilgen/IlBuilder.cpp

```diff
@@ -1193,26 +1194,36 @@ IlBuilder::operationWithOverflow(TR::ILOpCodes op, TR::Node *leftNode, TR::Node
    return resultValue;
    }
 
-TR::IlValue *
-IlBuilder::AddWithOverflow(TR::IlBuilder **handler, TR::IlValue *left, TR::IlValue *right)
+// This function takes 4 arguments and generate the addValue.
+// This function is called by AddWithOverflow and UnsignedAddWithOverflow.
+TR::ILOpCodes 
+IlBuilder::getOpCodeAndOperandNodes(TR::IlValue *leftValue, TR::IlValue *rightValue, TR::Node **leftNode, TR::Node **rightNode)
```

Comment:
This function, once it is streamlined to only compute the opcode, should also be shared with the Add() service, I think.